resharper/resharper-unity/src/Yaml/Psi/Modules/UnityExternalFilesPsiModule.cs

```diff
@@ -33,8 +33,6 @@ public class UnityExternalFilesPsiModule : UserDataHolder, IPsiModuleOnFileSyste
             Name = moduleName;
             TargetFrameworkId = targetFrameworkId;
             mySourceFiles = new CompactMap<FileSystemPath, Pair<IPsiSourceFile, LifetimeDefinition>>();
-            
-            PutData(DaemonEnablePolicy.EnableSwaForMiscModuleKey, true);
```

Comment:
I think this is why we had to have `IProjectFile` - `DaemonEnablePolicy.CalcSwaEnabled` only works on `IProjectFile` and if it's in the Misc Files project, its owning module requires this key set.